Makefile

```diff
@@ -46,7 +46,8 @@ JSFILES= \
 	assets/js/translator.js \
 	assets/js/analyzer.js \
 	assets/js/generator.js \
-	assets/js/sandbox.js
+	assets/js/sandbox.js \
+        assets/js/spellchecker.js
```

Comment:
Spacing

Reply:
@sushain97 this is weird, it looks fine in the actual file, but looks wrong here.